shared/settings/files/index.js

```diff
@@ -1,28 +1,110 @@
 // @flow
 import * as React from 'react'
 import * as Types from '../../constants/types/fs'
+import * as Constants from '../../constants/fs'
 import * as Kb from '../../common-adapters'
-import {fileUIName, isLinux} from '../../constants/platform'
+import {InlineDropdown} from '../../common-adapters/dropdown'
+import * as Platform from '../../constants/platform'
 import * as Styles from '../../styles'
 import SystemFileManagerIntegrationBanner from '../../fs/banner/system-file-manager-integration-banner/container'
 import RefreshDriverStatusOnMount from '../../fs/common/refresh-driver-status-on-mount'
+import RefreshSettings from './refresh-settings'
+import flags from '../../util/feature-flags'
 
 type Props = {|
+  areSettingsLoading: boolean,
   driverStatus: Types.DriverStatus,
-  onEnable: () => void,
   onDisable: () => void,
+  onEnable: () => void,
```

Comment:
I'm confused why you moved this down - if it's for alphabetical order you should probably swap `onEnableSyncNotifications` down too.